File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        int[] cnt = new int[26] ;
        for(char c : s.toCharArray()){
            cnt[c - 'a'] ++ ;
        }
        int ans = 0 ;
        for(int i = 0 ; i < 26 ; i ++)if(cnt[i] > 0){
            if(cnt[i] % 2 == 0) ans += 2 ;
            else ans += 1 ;
        }
        return ans ;
    }
}

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.py
class Solution:
    def minimumLength(self, s: str) -> int:
        cnt = [s.count(chr(ord('a') + i)) for i in range(26)]
        ans = 0 
        for i in cnt:
            if i > 0:
                ans += 1 if i % 2 != 0 else 2
        return ans 